ui/components/LoadingComponents/LoadingComponentServer.js

```diff
@@ -21,7 +23,10 @@ function LoadingScreen(props) {
         }}
       >
         <div>
-          <AnimatedLogoDark style={{ height: '100px', margin: '4px 0px 8px' }} />
+          <AnimatedLogoDark
+            style={{ height: '100px', margin: '4px 0px 8px' }}
+            textFill={theme.palette.type === 'dark' ? '#fff' : ''}
+          />
           <h1
             style={{
               fontFamily: 'sans-serif',
```

Comment:
Change to Sistent's theme text. Use Qanelas Soft here, please.

Reduce to an h2 or h3 - whatever is the size used when loading the final "Meshery Extensions" loading screen.

Reply:
Confirm that these settings are the same, @Pratik-050

Reply:
That’s not what I’m asking. There need to be changes in order for these two different loading screens to be the same. Confirm that there are no differences between the two, and if there are, make changes to make to align them.

Reply:
No sir everything is working fine, I'm just conditionally passing the color according to the theme and changed the div to the sistent typography component. That's all thanks!

Reply:
@Pratik-050, again, confirm the answers to my question regarding font family and font size.

Reply:
Hello sir, @leecalcote the font style is changed as said, and there is no field for font-size the token received from the sistent, the other component you've mentioned is using MUI, so I've made the font 16px that is h3 from mui library. Other than that we are OK! Thanks.